yt_dlp/options.py

```diff
@@ -408,6 +408,14 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         help=(
             'Location of the main configuration file; either the path to the config or its containing directory '
             '("-" for stdin). Can be used multiple times and inside other configuration files'))
+    general.add_option(
+        '--plugin-dirs',
+        dest='plugin_dirs', metavar='PATH', action='append',
+        help=(
+            'Path to a directory for yt-dlp to search for plugins (in addition to the default '
```

Comment:
I dont like this wording. How about sth like "path to an additional plugin directory"?

Reply:
Yeah we want to make sure that people know this doesn't overwrite the defaults but appends to the list of defaults.

`Path to a directory for yt-dlp to additionally search for plugins (in addition to the default plugin directories).`

Or

`Append a path to the list of default plugin directories yt-dlp uses to search for plugins.`

Reply:
Changed it to something that should be clear